### PR TITLE
Codestyle improvements (mostly sort imports)

### PR DESCRIPTION
**Pull request by mkoeppe**
Split out to preserve the incidental changes made in @tobiasdiez's original branch of #36372.

Author: Tobias Diez

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [ ] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by tobiasdiez — APPROVED** [review]
Thanks for helping upstreaming these changes.

Next time, please use the magic co-author comment to give proper attribution.

**Comment by mkoeppe** [comment]
Tobias, these are changes authored by you. You can't "positively review" them.

**Comment by mkoeppe** [comment]
I've rebased with --author

**Comment by tobiasdiez** [comment]
Okay, then I misunderstood the purpose of this PR. Could you then please review these changes? Thanks!

File: src/sage/rings/polynomial/pbori/fglm.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-from .PyPolyBoRi import Polynomial, BoolePolynomialVector
-from .pbori import FGLMStrategy, BooleSet
+from .pbori import BooleSet, FGLMStrategy
```
Comment (dcoudert) [inline src/sage/rings/polynomial/pbori/fglm.py:1]: We should avoid relative imports, no ?

**Review by dcoudert — CHANGES_REQUESTED** [review]
Why is it important to modify the order of the imports ? I'm missing something here.

File: src/sage/rings/polynomial/pbori/fglm.py (hunk 1 above)
Comment (tobiasdiez) [reply src/sage/rings/polynomial/pbori/fglm.py:1]: Yes, you are right. But I prefer to do this migration in a separate PR.

**Comment by tobiasdiez** [comment]
> Why is it important to modify the order of the imports ? I'm missing something here.

There is of course nothing important in the import order. As with any code style fix, it merely improves the readability of the code (slightly). For example, it's easier to spot related imports. Perhaps I should also add that these changes were done automatically by ruff.

**Comment by dcoudert** [comment]
Here also we have failing doctests due to nauty. I don't understand what's going on.

**Review by dcoudert — APPROVED** [review]
So then, LGTM.

**Comment by tobiasdiez** [comment]
Thanks for the review!

**Pull request merged**